### PR TITLE
Enhance savegame slot datetimes and hours played.

### PR DESCRIPTION
**Pull request by cbouwense**
### Issue \#
- None, I just thought I could make a small improvement.

### Changes Made
- Make "last played" and "time played" sections a little more human-readable.
- Renamed `formatTimeplayed` to `formatTimePlayed` to be consistent with camel-case style.
- Added a couple comments to logic sections. These can be removed, they were just to help me.

### Before
slot time formatting before

### After
slot time formatting after

### Testing
- Only manual visual testing. I could make automated tests for this if necessary!

### CODE REVIEW

File: apps/openmw/mwgui/savegamedialog.cpp [hunk 1]
```diff
@@ -402,32 +414,36 @@ namespace MWGui
...
-        std::stringstream text;
+        std::stringstream savegameMetadata;
+
+        // Add real world datetime for last played.
         time_t time = mCurrentSlot->mTimeStamp;
         struct tm* timeinfo;
         timeinfo = localtime(&time);
 
-        text << std::put_time(timeinfo, "%Y.%m.%d %T") << "\n";
+        savegameMetadata << std::put_time(timeinfo, "%b%e, %Y %r %p") << "\n";
```
Comment (cbouwense) [inline apps/openmw/mwgui/savegamedialog.cpp:424]: interestingly, `%b` (abbreviated month name) seems to have a space after it, so no space is needed here between `%b` and `%e`

**Comment by psi29a** [comment]
What if people prefer ISO standard readout. :)

**Comment by cbouwense** [comment]
Perhaps I could make this a toggleable setting. I guess I just assumed everyone would rather have this haha

**Comment by cbouwense** [comment]
@psi29a I saw your comment on the forums about this being English-centric, and I hadn't considered that! It's a good point. 

I wonder if this change would make sense if I scope it just to the English game setting?

**Comment by psi29a** [comment]
I think it's something we should discuss technically in the forums first. To see if there is a better way to handle this. Perhaps this is something that will be part of the UI dehardcoding that would allow mods (Lua) to change. I'm not saying what you're doing is right or wrong, just that we need more input before continuing.

**Comment by AnyOldName3** [comment]
It's specifically US-centric. None of the world's many other English-speaking locales use inside out ordering (which is sensible as the US format is insane).

**Comment by cbouwense** [comment]
Though I disagree that this change is "insane", it appears this format is unwanted by the team members who have commented here and [on the forums](https://forum.openmw.org/viewtopic.php?p=72308). Closing this PR

**Comment by tauin** [comment]
there are other countries that use the 12 hr format commonly (like australia for example) but i agree that this should be done using lua instead

**Comment by AnyOldName3** [comment]
Twelve hour isn't the thing that's especially objectionable as that's normal in a lot of the world. It's just the US that does Month Day Year instead of Day Month Year or Year Month Day, though.

**Pull request closed without merging**